docs/phylum_group.md

```diff
@@ -0,0 +1,28 @@
+---
+title: phylum group
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Interact with user groups
+
+```sh
+phylum group [OPTIONS] [SUBCOMMAND]
+```
+
+### Options
+`-j`, `--json`
```

Comment:
This isn't clear to me. Is `--json` a parameter to `phylum group`, or is it a parameter to `phylum group list`, which is the default subcommand of `phylum group`? If I run `phylum group -j create sample`, will that produce output in JSON?

Reply:
`--json` is a parameter to `phylum group` command (see line 170)

![image](https://user-images.githubusercontent.com/34108612/169090774-3c6a9eb2-dc3a-43bd-ba98-85aaffeb02c5.png)